docs/update_and_migration/from_4.6/update_from_4.6.md

```diff
@@ -431,6 +431,27 @@ No additional steps needed.
 
 With the product updated to the latest version, you can now finish the update process or proceed to updating the LTS Updates packages.
 
+## v4.6.25
+
+### Form Builder performance fix: missing indexes on form submission data
+
+In large production databases, the `ezform_form_submission_data` table may contain a lot of rows.
```

Comment:
Two tables are touched, ezform_form_submissions and ezform_form_submission_data. If you mentioned one, you should maybe name the other too?